### ISSUE TITLE
Pretrained model on KITTI-360

### ISSUE THREAD

**Issue by ngoductuanlhp** [cv-rits/MonoScene#13]
Hi @anhquancao ,

Thanks so much and congratulate you on the acceptance of your paper at CVPR2022. Do you try to train your model on the KITTI-360 dataset? I want to evaluate the result on KITTI-360, have you ever tried to generate the SSC groundtruth of KITTI-360?

Thanks.

**Comment by anhquancao** [comment]
Hi @ngoductuanlhp
Thank you for your kind words! We didn't train MonoScene on the KITTI-360 dataset since there is no SSC ground-truth available.
We didn't want to generate the SSC ground truth by ourselves since it might be inconsistent with the official dataset.

**Comment by ngoductuanlhp** [comment]
Hi @anhquancao , I have tried to generate the SSC ground truth for KITTI-360 to train your model on this dataset. However, the loss is very weird. Do you have any suggestions for training on KITTI-360, which config do I need to modify? And could you provide the log train of your model on semantic-KITTI?

Thansks

**Comment by anhquancao** [comment]
You can take a look at this data loader for the kitti_360 dataset
https://github.com/anhquancao/xmuda-extend/blob/master/xmuda/data/kitti_360/kitti_360_dataset.py

For the log, I uploaded them to this link
https://tensorboard.dev/experiment/NFmpSQQRREub4wVwRyQMKw/#scalars&_smoothingWeight=0
